docs/Deploying/FAQ.md

```diff
@@ -234,3 +234,6 @@ If you would like to manage hosts that can travel outside your VPN or intranet w
 ## What is the minimum version of MySQL required by Fleet?
 
 Fleet requires at least MySQL version 5.7.
+
+## How do I migrate from Fleet Free to Fleet Premium
```

Comment:
Looks like you'll want a `?` at the end of the heading for consistency

Reply:
Yes of course. Thanks @mikermcneil